.github/CODEOWNERS

```diff
@@ -989,6 +989,13 @@
 # ServiceLabel: %Spring App Discovery %Mgmt
 # ServiceOwners:                                                   @sunkun99
 
+# PRLabel: %Informaticadatamanagement
```

Comment:
```suggestion
# PRLabel: %Informatic Data Management
```

You cannot create arbitrary new labels.   Please use "Informatic Data Management", which I've created on your behalf.

Reply:
@jsquire I am assuming you have created the new label as _"Informatica Data Management"_, and not as _"Informatic Data Management"_

Reply:
Yes.  that's a typo.   Apologies.